docs/INSTALL.md

```diff
@@ -111,14 +111,14 @@ brew install cmake qt@5
 
 ### Building
 
-To build a debug version of the desktop app:
+To build a desktop app:
```

Comment:
(use -d for debug)

Reply:
Its there in the
> Check `tools/unix/build_omim.sh -h` for more build options, e.g. to build a debug version.
already.

The reason I changed the default build from debug to release is that most users will want to just build and run it, the debug version is needed by devs mostly. 

Also debug versions of binaries take crazy amount of disk space, e.g. the release `OMaps` is 21Mb while the debug OMaps is 400Mb+. I was trying to build debug versions of all targets first and ran out of disk space just when the build dir reached 12Gb :(